### PR TITLE
[PCFG-82] SIA Database configuration

### PR DESCRIPTION
**Pull request by jacopocarlini**
### List of changes

- added configuration to `apiconfig` for SIA DB
- added read-only policy to API

### Motivation and context

To set the configuration for dev, uat, prod

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/api/apiconfig_api/v1/_openapi.json.tpl [hunk 1]
```diff
@@ -1,2049 +1,2710 @@
 {
-  "openapi": "3.0.1",
+  "swagger": "2.0",
```
Comment (pasqualespica) [inline src/api/apiconfig_api/v1/_openapi.json.tpl:2]: why this ?

File: src/env/dev/terraform.tfvars [hunk 2]
```diff
@@ -202,6 +202,6 @@ acr_enabled = true
 
 # db nodo dei pagamenti
 db_port         = 1523
-db_service_name = "" # fixme set with data from SIA
+db_service_name = "NDPSPCT_NODO4_CFG"
```
Comment (pasqualespica) [inline src/env/dev/terraform.tfvars:205]: comment with `fixme ask to SIA`

File: src/env/prod/terraform.tfvars [hunk 3]
```diff
@@ -199,6 +199,6 @@ eventhubs = [
 ]
 
 # db nodo dei pagamenti
-db_service_name = "" # fixme set with data from SIA
+db_service_name = "NDPSPCP_NODO4_CFG"
```
Comment (pasqualespica) [inline src/env/prod/terraform.tfvars:202]: ditto

File: src/env/uat/terraform.tfvars [hunk 4]
```diff
@@ -200,6 +200,6 @@ eventhubs = [
 ]
 
 # db nodo dei pagamenti
-db_service_name = "" # fixme set with data from SIA
+db_service_name = "NDPSPCA_NODO4_CFG"
```
Comment (pasqualespica) [inline src/env/uat/terraform.tfvars:203]: ditto

File: src/api/apiconfig_api/v1/_base_policy.xml [hunk 5]
```diff
@@ -12,15 +12,23 @@
 -->
 <policies>
     <inbound>
-        <base />
+        <base/>
+        <set-variable name="isGet" value="@(context.Request.Method.Equals("GET"))" />
+        <choose>
+            <when condition="@(context.User.Groups.Any(g => g.Name == "Read Only") && !context.Variables.GetValueOrDefault<bool>("isGet"))">
```
Comment (pasqualespica) [inline src/api/apiconfig_api/v1/_base_policy.xml:18]: where is defined [read only group](https://portal.azure.com/?l=en.en-us#blade/Microsoft_Azure_ApiManagement/GroupBlade/groupOverview/id/%2Fsubscriptions%2Fbbe47ad4-08b3-4925-94c5-1278e5819b86%2FresourceGroups%2Fpagopa-d-api-rg%2Fproviders%2FMicrosoft.ApiManagement%2Fservice%2Fpagopa-d-apim%2Fgroups%2Fread-only/name/Read%20Only) ?

**Review by pasqualespica — CHANGES_REQUESTED** [review]

File: src/api/apiconfig_api/v1/_base_policy.xml (hunk 5 above)
Comment (jacopocarlini) [reply src/api/apiconfig_api/v1/_base_policy.xml:18]: added terreform definition for [read-only group ](https://github.com/pagopa/pagopa-infra/pull/64/files#diff-436422a5916b2158974132848de0a192bc0e9df0b06d9d0f776594540ac9f7cdR73-R80)

**Review by pasqualespica — APPROVED** [review]
🚀

**Review by pasqualespica — APPROVED** [review]

**Pull request merged**